### PR TITLE
Split the requirements files for each enviroment

### PR DESCRIPTION
**Pull request by oscarpolanco**
Closes [Ticket](https://trello.com/b/7te0MkIv/ddj-neo4j)

Split the requirement file to install only the dependency we want for enviroment

## Testing

Steps to manually verify the change:
1. _Setup_: set your virtual env
2. _Exercise_: Run `pip install -r requirements/dev.txt` and `pip install -r requirements/production.txt`
3. _Verify_: See that every dependency that you need is install

### CODE REVIEW

File: paradise_papers_search/requirements/dev.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+-r base.txt
```
Comment (oscarpolanco) [inline paradise_papers_search/requirements/dev.txt:1]: Add this file in case we need a debug dependency locally. What you think? Should I delete this?  🤔

Comment (raulgomezc) [reply paradise_papers_search/requirements/dev.txt:1]: Leave it, not gonna hurt anyone to have it there

**Review by raulgomezc — APPROVED** [review]
LGTM

**Review by mariamera — APPROVED** [review]
👍

**Comment by junior1q94** [comment]
👍

**Pull request merged**